script/.env

```diff
@@ -11,4 +12,4 @@ IOS_DESTINATION_VERSION=${IOS_DESTINATION_VERSION:-"latest"}
 IOS_DESTINATION_SIMULATOR_NAME=${IOS_DESTINATION_SIMULATOR_NAME:-"iPhone 6"}
 OSX_DESTINATION_ARCH=${OSX_DESTINATION_ARCH:-""}
 INSTALL_GITHOOKS=true
-REQUIRED_XCODE_VERSION=7.3.1
+FORCE_SWIFTLINT_VERSION=true
```

Comment:
Great. The same happened to me with carthage in git hooks and it was annoying because I had two projects with different carthage versions. Maybe we could change that too?